docker/jdk9/x86_64/ubuntu/Dockerfile

```diff
@@ -40,8 +39,18 @@ RUN apt-get update \
     git \
     unzip \
     realpath \
+    g++-4.8 \
+    gcc-4.8 \
 && rm -rf /var/lib/apt/lists/*
 
+# Make sure build uses 4.8
+RUN rm /usr/bin/gcc \
+    && rm /usr/bin/g++ \
```

Comment:
will this fail if g++ doesn't already exist? perhaps add `-f` to be safe

Reply:
the base image installs g++, so the original is there "by construction"